pkg/services/shorturls/short_url_service.go

```diff
@@ -76,3 +76,16 @@ func (s ShortURLService) CreateShortURL(ctx context.Context, user *models.Signed
 
 	return &shortURL, nil
 }
+
+func (s ShortURLService) DeleteStaleShortURLs(ctx context.Context, cmd *models.DeleteShortUrlCommand) error {
+	return s.SQLStore.WithTransactionalDbSession(ctx, func(session *sqlstore.DBSession) error {
+		var rawSql = "DELETE FROM short_url WHERE created_at <= ? AND last_seen_at IN (NULL, 0)"
```

Comment:
I think it's safest to check `NULL` and `0` as the column is nullable, but will seem to default to 0 (confirmed on all the different DBs). So since technically it could be both, I think this is the best approach

Reply:
Make sense. Weird though it is nullable, didn't think it was.